### PR TITLE
New tool based on polyscope: polyMeshEdit

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description
  polyMeshEdit: new tool to edit a mesh (add local noise remove selected faces). In the same way than meshViewerEdit based on Qt  but using Polyscope and PolygonalSurface. 

Capture d’écran 2023-11-17 à 17 47 26

# Checklist

- [x] Doxygen documentation of the code completed (classes, methods, types, members...)
- [x] Check if it follows the tools structure described in [CONTRIBUTING.md](https://github.com/DGtal-team/DGtalTools-contrib/blob/master/CONTRIBUTING.md)
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtalTools-contrib/blob/master/ChangeLog.md) added.
- [x] Update the readme with potentially a screenshot of the tools if it applies. 
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [x] All continuous integration tests pass (Github Actions & appveyor).

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -15,8 +15,12 @@
  - *Geometry3d*
   - basicMorphoFilter: fix a bug on the dilate/erode options. 
     (Bertrand Kerautret [#74](https://github.com/DGtal-team/DGtalTools-contrib/pull/74))  
-  
- # DGtalTools-contrib  1.3 
+ - *visualisation*
+  - polyMeshEdit: tool to edit a mesh (add local noise remove selected faces).
```
Comment (dcoeurjo) [inline ChangeLog.md:19]:
```suggestion
  - polyMeshEdit: tool to edit a mesh (add local noise, remove selected faces).
```

File: README.md [hunk 2]
```diff
@@ -91,8 +91,12 @@ This section, can contain all tools related to visualisation:
    - displayTgtCoverAlphaTS: to display alpha-thick segment given on a simple contour.
    - meshViewerEdit: tool to visualize a mesh and to apply simple edits (face removal, color edits...).
    - graphViewer: tool to display graphs from a list of edges, a list of vertex and an optionnal list of radius for each edge.
+   - polyMeshEdit: tool to edit a mesh (add local noise remove selected faces).
```
Comment (dcoeurjo) [inline README.md:94]:
```suggestion
   - polyMeshEdit: tool to edit a mesh (add local noise, remove selected faces).
```

File: visualisation/polyMeshEdit.cpp [hunk 3]
```diff
@@ -0,0 +1,417 @@
...
+///////////////////////////////////////////////////////////////////////////////
+using namespace std;
+using namespace DGtal;
+///////////////////////////////////////////////////////////////////////////////
+
+
+/**
+ @page polyMeshEdit polyMeshEdit
+ 
+ @brief  polyMeshEdit tools to edit a mesh (add local noise remove selected faces). Note that the process rely on half edge data structure that can fails if the input is not topological consistant. If you want use other type of mesh, you can use meshViewerEdit that is based on the simple soup of triangles process (slower selection process).
```
Comment (dcoeurjo) [inline visualisation/polyMeshEdit.cpp:56]:
```suggestion
 @brief  polyMeshEdit tools to edit a mesh (add local noise and remove selected faces). Note that the process relies on the halfedge data structure that can fail if the input is not topologically consistant. If you want use other type of mesh, you can use meshViewerEdit that is based on the simple soup of triangles process (slower selection process).
```

File: visualisation/polyMeshEdit.cpp [hunk 4]
```diff
@@ -0,0 +1,417 @@
...
+
+
+
+int main(int argc, char** argv)
+{
+    std::string inputFileName {""};
+    
+    // parse command line using CLI ----------------------------------------------
+    CLI::App app;
+    app.description("polyMeshEdit tool to edit a mesh (add local noise remove selected faces). Note that the process rely on half edge data structure that can fails if the input is not topological consistant. If you want use other type of mesh, you can use meshViewerEdit that is based on the simple soup of triangles process (slower selection process). \n"
```
Comment (dcoeurjo) [inline visualisation/polyMeshEdit.cpp:384]:
```suggestion
    app.description("polyMeshEdit tool to edit a mesh (add local noise and remove selected faces). Note that the process relies on the halfedge data structure that can fail if the input is not topologically consistant. If you want use other type of mesh, you can use meshViewerEdit that is based on the simple soup of triangles process (slower selection process). \n"
```

**Review by dcoeurjo — APPROVED** [review]
Nice PR!
Just few comments.

**Comment by kerautret** [comment]
Thanks for the suggest, just done

**Pull request merged**